commercetools-models/src/main/java/io/sphere/sdk/categories/commands/updateactions/SetAssetCustomType.java

```diff
@@ -17,21 +17,35 @@
  */
 public final class SetAssetCustomType extends SetCustomTypeBase<Category> {
     private final String assetId;
+    private final String assetKey;
 
-    private SetAssetCustomType(final String assetId, @Nullable final String typeId, @Nullable final String typeKey, @Nullable final Map<String, JsonNode> fields) {
+    private SetAssetCustomType(final String assetId, final String assetKey, @Nullable final String typeId, @Nullable final String typeKey, @Nullable final Map<String, JsonNode> fields) {
         super("setAssetCustomType", typeId, typeKey, fields);
         this.assetId = assetId;
+        this.assetKey = assetKey;
     }
 
     public String getAssetId() {
         return assetId;
     }
 
+    public String getAssetKey() {
+        return assetKey;
+    }
+
     public static SetAssetCustomType of(final String assetId, @Nullable final CustomFieldsDraft customFieldsDraft) {
+        return of(assetId,null, customFieldsDraft);
+    }
+
+    public static SetAssetCustomType ofKey(final String assetKey, @Nullable final CustomFieldsDraft customFieldsDraft) {
+        return of(null,assetKey, customFieldsDraft);
+    }
+
+    private static SetAssetCustomType of(final String assetId,final String key, @Nullable final CustomFieldsDraft customFieldsDraft) {
```

Comment:
For consistency, I would suggest to rename the `key` parameter to `assetKey` here.

Reply:
good catch 👍